language/vm/src/deserializer.rs

```diff
@@ -1227,7 +1227,7 @@ impl Opcodes {
             0x30 => Ok(Opcodes::IMM_BORROW_GLOBAL),
             0x31 => Ok(Opcodes::MOVE_FROM),
             0x32 => Ok(Opcodes::MOVE_TO),
-            0x33 => Ok(Opcodes::CREATE_ACCOUNT),
+            0x33 => Ok(Opcodes::CREATE_ACCOUNT_DEPRECATED),
```

Comment:
Personally I like replacing this line with
`0x33 => Err(deprecated_bytecode)`
And we can batch remove all the deprecated opcode at once in the future?

Reply:
it seems we are pushing for the change to go in as is, given it's not breaking existing scripts.
I may get a conversation going to find out whether we have consensus or not.
To be clear the idea of pushing the change as is is related to the fact the opcode is high enough in the list not to affect existing scripts.
If existing scripts where affected I would definitely agree with you.
This way I am not too clear what is best.
Do you have some scenario that we would break and we should be concerned about?
I would definitely want to hear them.

Reply:
@runtian-zhou if you did not, take a look at #1594 and see if that makes sense to you.
Otherwise let me know your thoughts as to the above and as I said I will start a conversation later about this